src/main/java/teammates/common/util/Const.java

```diff
@@ -386,7 +386,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String FEEDBACK_QUESTION_GETLINK =
                 "Get a submission link to this particular question. "
                 + "Useful if you want students to answer individual questions separately or at different points in time.";
-        public static final String FEEDBACK_QUESTION_CANCEL = "Discard your changes";
+        public static final String FEEDBACK_QUESTION_DISCARDCHANGES = "Discard your changes";
```

Comment:
@damithc I think we should use more of "you/your", since most of the UIX messages involves giving instructions/information to the user.

The [Microsoft Design Guidelines] (https://msdn.microsoft.com/en-us/library/dn742477.aspx#person) provides relevant guidance: specifically, use "you/your" when giving instructions, and use "I/my" when accepting instructions from the user.

But there may still be places where "I/my" is more applicable: e.g., on most log-in pages, there is "by clicking on this... __you__ agree to..." and also the option to "remember __me__" upon logging in

For now, one thing to standardize will be to rephrase the questions in Team Peer Evaluation Sessino to use "you/your"